### ISSUE TITLE
[WIP]Console logs are not WCAG contrast compliant

### ISSUE THREAD

**Issue by colinhemphill** [dequelabs/react-axe#100]
This may seem petty, but I think there's a good case for "practice what you preach".

### Moderate warnings: #ffa500
Contrast ratio of 1.97 on white background — fails all compliance tests.

_Suggested change:_
Chrome uses **#5c3c00** as dark text on a **#fffbe5** yellow background, which is AAA compliant.

<img width="105" alt="Screen Shot 2019-07-16 at 4 27 41 PM" src="https://user-images.githubusercontent.com/1854690/61331039-adb2be80-a7e6-11e9-9d29-f40fdfcd6413.png">

In other places they use a similar gold color on a white background:

<img width="160" alt="Screen Shot 2019-07-16 at 4 26 49 PM" src="https://user-images.githubusercontent.com/1854690/61330985-8bb93c00-a7e6-11e9-9d4a-05271bde8ea0.png">

### Severe warnings: #ff0000
Contrast ratio of 4.0 on white background — only AA compliant at large sizes. This one isn't as egregious as the warnings.

_Suggested change:_
Chrome uses **#c51916** for red text, which is AAA compliant. They also use #ff0000 for error logs in some places, so ¯\_(ツ)_/¯

**Comment by WilcoFiers** [comment]
@colinhemphill It's more of a damned if you do, damned if you don't type of situation. If we up the contrast for a light theme, it's lowered for a dark theme, and the dark theme isn't dark enough that there's a color that meets the requirements in both themes.

**Comment by missmatsuko** [comment]
Can't the background color be set so that the text to background contrast can be controlled? Only thing is it may still clash with the browser theme. Not sure if there's a way to read the user's light/dark theme preference that makes sense for this. Maybe configuration is still a better option.

**Comment by straker** [comment]
QA: Test color ratios in both light and dark themes

**Comment by padmavemulapati** [comment]
Below are the light and dark mode theams observed, the color ratio observed 4.5:1 - for  light background observing the color pallatte as serious: '#d93251', minor: '#d24700', text: 'black'
And 
for dark background -  serious: '#ffb3b3',  minor: '#ffd500',  text: 'white'
![image (8)](https://user-images.githubusercontent.com/47409579/76750638-a2b09380-67a4-11ea-8d04-aca6c836f377.png)
![image (9)](https://user-images.githubusercontent.com/47409579/76750657-a7754780-67a4-11ea-8885-6d798fb329de.png)
